run.py

```diff
@@ -149,17 +149,25 @@ def cleanup(dicom_directory, output_dir):
         if os.path.isfile(full_file_name):
 
             if os.path.splitext(Path(file_name))[1] == '.dcm':
-                if not os.path.exists(dicom_directory / Path('dcms_to_zip')):
-                    os.mkdir(dicom_directory / Path('dcms_to_zip'))
-                dest = dicom_directory / Path('dcms_to_zip') / Path(file_name)
+                if not os.path.exists(dicom_directory / 'dcms_to_zip'):
+                    os.mkdir(dicom_directory / 'dcms_to_zip')
+                dest = dicom_directory / 'dcms_to_zip' / Path(file_name)
 
             else:
                 dest = output_dir / Path(file_name)
 
             os.rename(full_file_name, dest)
 
     col = DICOMCollection.from_dir(dicom_directory / Path('dcms_to_zip'))
-    col.to_zip(output_dir / Path('results.zip'))
+
+    split_1 = str(context.get_input_path("Input_file")).split('.svs')
+    split_2 = str(Path(context.get_input_path("Input_file")).stem).split('.')
+    if len(split_1) > 1:
+        output_stem = Path(split_1[0]).stem
+    else:
+        output_stem = split_2[0]
```

Comment:
why not a plain `Path(context.get_input_path("Input_file")).stem` ?

Input_file can be 10 different file types if I trust the readme